PurchasesCoreSwift/Logging/Strings/ConfigureStrings.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  ConfigureStrings.swift
+//  PurchasesCoreSwift
+//
+//  Created by Tina Nguyen on 12/11/20.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+import Foundation
+
+@objc(RCConfigureStrings) public class ConfigureStrings: NSObject {
+    @objc public var adsupport_not_imported: String { "AdSupport framework not imported. Attribution data incomplete." } //warn
+    @objc public var application_active: String { "applicationDidBecomeActive" } //debug
+    @objc public var configuring_purchases_proxy_url_set: String { "Purchases is being configured using a proxy for RevenueCat with URL: %@" } //info
+    @objc public var debug_enable: String { "Debug logging enabled" } //debug
+    @objc public var delegate_set: String { "Delegate set" } //debug
+    @objc public var duplicate_purchases_instance: String { "Purchases instance already set. Did you mean to configure two Purchases objects?" } //info
```

Comment:
this isn't accurate - there will still only be one instance unless the customer is holding on to the former instance, which is unlikely, so it will probably not be a duplicate. 
maybe
`purchases_instance_already_set`